crypto/pem/pem_lib.c

```diff
@@ -537,12 +536,13 @@ int PEM_get_EVP_CIPHER_INFO(char *header, EVP_CIPHER_INFO *cipher)
     *header = '\0';
     cipher->cipher = enc = EVP_get_cipherbyname(dekinfostart);
     *header = c;
+    header++;
```

Comment:
And I look at the current code and say why not a single statement.  Shrug.  I'll do it myself then.  It will take a bit longer.